test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,46 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertNotNull();
+
+        // only one object
+        assertNull((Object) null);
+        assertNotNull(1);
+        assertNotNull("");
+        assertNotNull("abc");
+
+        // all objects null
+        assertNull(null, null, null);
+        assertNull(null, null);
+
+        // some objects null
+        assertNull(null, 1, new Integer(1));
+        assertNull(null, null);
+        assertNull(null, "a", "b", null);
+
+        // no objects null
+        assertNotNull("abc", "abc");
+        assertNotNull("abc", "", "abc", "ABC");
+        assertNotNull("", "abc", "a", "abc");
+        assertNotNull(1, new Integer(1));
```

Comment:
Quite thorough, however many redundancy.
Does each additional test case have chance of finding a new bug?